src/d/actor/d_a_tag_attention.cpp

```diff
@@ -37,42 +39,118 @@ static dCcD_SrcSph sph_check_src = {
     },
 };
 
+bool daTagAttention::Act_c::chk_inside(cXyz* pos) const {
+    if (!m_b0x290){
+        return false;
+    }
+
+    cXyz plyrToObjVec = dComIfGp_getPlayer(0)->current.pos - current.pos;
+    if (subtype == 0){
+        // spherical collision check
+        f32 distance = plyrToObjVec.abs();
+        
+        if (distance > scale.x * 100.0f) {
+            return false;
+        }
+    }else{
+        // box collision check
+        s16 yRotAngle = current.angle.y;
+        if (yRotAngle != 0){
+            plyrToObjVec.z = 
+                plyrToObjVec.x * cM_ssin(yRotAngle) +
+                plyrToObjVec.z * cM_scos(yRotAngle);
+            plyrToObjVec.x = 
+                plyrToObjVec.x * cM_scos(yRotAngle) -
+                plyrToObjVec.z * cM_ssin(yRotAngle);
+        }
+
+        f32 curScale = scale.x;
+        if ((plyrToObjVec.x < -curScale * 100.0f) || (curScale * 100.0f > plyrToObjVec.x)){
+            return false;
+        }
+        curScale = scale.y;
+        if ((plyrToObjVec.y < -curScale * 100.0f) || (curScale * 100.0f > plyrToObjVec.y)){
+            return false;
+        }
+        curScale = scale.z;
+        if ((plyrToObjVec.z < -curScale * 100.0f) || (curScale * 100.0f > plyrToObjVec.z)){
+            return false;
+        }
+    }
+
+    *pos = current.pos;
+    return true;
+}
 
 /* 00000078-00000188       .text _create__Q214daTagAttention5Act_cFv */
 cPhs_State daTagAttention::Act_c::_create() {
-    /* Nonmatching */
+    fopAcM_SetupActor(this, Act_c);
+    mStts.Init(0xFF, 0xFF, this);
+    mSph.Set(sph_check_src);
+    mSph.SetStts(&mStts);
+    return cPhs_COMPLEATE_e;
 }
 
 /* 00000340-00000480       .text _execute__Q214daTagAttention5Act_cFv */
 bool daTagAttention::Act_c::_execute() {
     /* Nonmatching */
+    mSph.SetC(current.pos);
+    mSph.SetR(scale.x * 100.0f);
+    dComIfG_Ccsp()->Set(&mSph);
+    m_b0x290 = true;
+    int iVar1 = prm_get_Type();
+    if (iVar1 == 1){
+        iVar1 = prm_get_swSave();
+        BOOL bVar2 = dComIfGs_isSwitch(iVar1, home.roomNo);
+        if (bVar2 == FALSE){
+            m_b0x290 = false;
+        }
+    }else{
+        iVar1 = prm_get_Type();
+        if (iVar1 == 2){
+            iVar1 = prm_get_swSave();
+            BOOL bVar2 = dComIfGs_isSwitch(iVar1, home.roomNo);
+            if (bVar2 == FALSE){
+                m_b0x290 = false;
+            }
+        }
+    }
+
+    // TODO for matching: implement chk_inside and dComIfGp_att_Look2RequestF
+    bool wasHit = chk_inside(&current.pos);
+    if (wasHit){
```

Comment:
```suggestion
    cXyz unused;
    if (chk_inside(&unused)){
```
For this to match, you need to pass an uninitialized `cXyz` into the function